src/ngraph/frontend/onnx_import/onnx.cpp

```diff
@@ -35,15 +35,15 @@ namespace ngraph
                 struct file_open : ngraph_error
                 {
                     explicit file_open(const std::string& path)
-                        : ngraph_error{"failure opening file:" + path}
+                        : ngraph_error{"Failure opening file: " + path}
                     {
                     }
                 };
 
                 struct stream_parse : ngraph_error
                 {
                     explicit stream_parse(std::istream&)
-                        : ngraph_error{"failure parsing data from the stream"}
+                        : ngraph_error{"Failure parsing data from the stream"}
```

Comment:
Let's try to make this message a bit easier to understand. For example: change `the stream` to `input stream`